OpenSim/Simulation/Model/BushingForce.h

```diff
@@ -80,15 +75,28 @@ OpenSim_DECLARE_CONCRETE_OBJECT(BushingForce, Force);
 //==============================================================================
 // PUBLIC METHODS
 //==============================================================================
-    /** Default constructor leaves bodies unspecified, sets the bushing frames
+    /** Default constructor leaves frames unspecified, sets the bushing frames
     to be at their body origins, and sets all bushing parameters to zero. **/
     BushingForce();
-    /** This constructor provides everything needed to create a real 
-    bushing. See property declarations for more information. **/
-    BushingForce(const std::string& body1Name, 
+
+    /** Construct the BushingForce given the offset frames that it tries to
+    align according to the physical properties of the bushing.
+    See property declarations for more information. **/
+    BushingForce(const PhysicalOffsetFrame& frame1,
```

Comment:
It is starting to sound valuable to not make a component like BushingForce choose between "ownership" OR "reference". Ideally, if the user wanted to refer to a body for the PhysicalFrame, they could; but if they wanted to directly insert a new PhysicalOffsetFrame that would be managed by BushingForce, they could do that too.

This specific issue came up with the pseudocde session. I'd like to do either of the following:

```
<components>
    <data_to_track>
        <DataSource> <filename>mydata.csv</filename></DataFile>
    </data_to_track>
</components>
```
(DataSource owned by data_to_track)

-or-

```
<components>
    <data_to_track ref="mydata"/>
    <DataSource name="mydata"> <filename>mydata.csv</filename></DataFile>
<components>
```
(DataSource is simply referenced)

This resembles part of what @aymanhab once suggested for the xml related to connectors.

I think forcing to choose either ownership or reference may put us in a lot of undesirable design corners. Perhaps we could have the notion of a "ReferenceProperty".

I don't think this is a topic to consider within this PR though.